src/constants.js

```diff
@@ -17,16 +17,54 @@ Sk.builtin.bool.false$ = /** @type {Sk.builtin.bool} */ (Object.create(Sk.builti
 /* Constants used for kwargs */
 
 // Sk.builtin.int_
-Sk.builtin.int_.co_varnames = [ "base" ];
-Sk.builtin.int_.co_numargs = 2;
-Sk.builtin.int_.$defaults = [ new Sk.builtin.int_(10) ];
+Sk.builtin.int_.co_varnames = [ "number", "base" ];
+Sk.builtin.int_.$defaults = [ Sk.builtin.none.none$ ];
 
 // Sk.builtin.lng
-Sk.builtin.lng.co_varnames = [ "base" ];
-Sk.builtin.lng.co_numargs = 2;
-Sk.builtin.lng.$defaults = [ new Sk.builtin.int_(10) ];
+Sk.builtin.lng.co_varnames = [ "number", "base" ];
+Sk.builtin.lng.$defaults = [ Sk.builtin.none.none$ ];
 
 // Sk.builtin.sorted
-Sk.builtin.sorted.co_varnames = ["cmp", "key", "reverse"];
-Sk.builtin.sorted.co_numargs = 4;
+Sk.builtin.sorted.co_varnames = ["list", "cmp", "key", "reverse"];
 Sk.builtin.sorted.$defaults = [Sk.builtin.none.none$, Sk.builtin.none.none$, Sk.builtin.bool.false$];
+
+Sk.builtin.int_.co_name = new Sk.builtin.str("int");
+Sk.builtin.lng.co_name = new Sk.builtin.str("long");
+Sk.builtin.sorted.co_name = new Sk.builtin.str("sorted");
+Sk.builtin.range.co_name = new Sk.builtin.str("range");
+Sk.builtin.round.co_name = new Sk.builtin.str("round");
+Sk.builtin.len.co_name = new Sk.builtin.str("len");
+Sk.builtin.min.co_name = new Sk.builtin.str("min");
+Sk.builtin.max.co_name = new Sk.builtin.str("max");
+Sk.builtin.sum.co_name = new Sk.builtin.str("sum");
+Sk.builtin.zip.co_name = new Sk.builtin.str("zip");
+Sk.builtin.abs.co_name = new Sk.builtin.str("abs");
+Sk.builtin.abs.co_name = new Sk.builtin.str("fabs");
```

Comment:
Should this be `.fabs`?

Maybe this could be refactored into an array of names and a for each?  Might make it more maintainable if `builtindict.js` could run off of that array as well.